trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/extract-query-and-session-data.test.ts

```diff
@@ -91,12 +93,12 @@ describe('extractQueryAndSessionData', () => {
   it('uses selectedFeeRecordIds from session data for isCheckboxChecked when present', () => {
     // Arrange
     const queryParams = {
-      facilityIdQuery: FACILITY_ID_QUERY,
+      premiumPaymentsFacilityId: PREMIUM_PAYMENTS_FACILITY_ID_QUERY,
     };
     const sessionData = {};
 
     jest.mocked(handleRedirectSessionData).mockReturnValue({
-      tableDataError: undefined,
+      premiumPaymentsTableDataError: undefined,
```

Comment:
Same as above, can we have an optional type instead of having to pass `undefined` values around?